### PR TITLE
chore(scheduler): cleanup some minor issues in testing

### PR DESCRIPTION
**Pull request by Abirdcfly**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func TestMostAllocatedScoringStrategy(t *testing.T) {
 		{Name: string(v1.ResourceMemory), Weight: 1},
 	}
 	extendedRes := "abc.com/xyz"
-	extendedResourceLeastAllocatedSet := []config.ResourceSpec{
+	extendedResourceMostAllocatedSet := []config.ResourceSpec{
```
Comment (denkensk) [inline pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go:40]: I find it's independent of the score algorithms and is used both in `TestLeastAllocatedScoringStrategy` and `TestMostAllocatedScoringStrategy` with different names.
Maybe we can define it separately in `test_util.go` and use it elsewhere? WDYT?

**Review by denkensk — COMMENTED** [review]
Thanks @Abirdcfly  Small comment.

**Comment by denkensk** [comment]
/lgtm

Thanks @Abirdcfly

**Comment by Huang-Wei** [comment]
@Abirdcfly would you mind retitling the issue to indicate it's a cleanup instead of issue fixing?

**Comment by Huang-Wei** [comment]
/approve

/hold
(feel free to cancel it after retitling the PR)

**Comment by Abirdcfly** [comment]
> retitling the issue to indicate it's a cleanup instead of issue fixing?

Thanks. @Huang-Wei 

The pr and commit messages have been updated.

just commit message update, no change in content: https://github.com/kubernetes/kubernetes/compare/e83fb507080318e5660292f8debd0b1d0bdf1291..bf8570029e680231372ad1382d23b4ffa8dfa8ae

(Surprised to see that `/lgtm` is cancelled after force-push, but `/approve` is not)

**Comment by Huang-Wei** [comment]
/lgtm
/unhold

Thanks @Abirdcfly .

**Pull request merged**